components/navbar/ProfileDropdown.vue

```diff
@@ -165,7 +165,7 @@
         class="button-connect-wallet px-4"
         variant="k-accent"
         no-shadow
-        @closeBurgerMenu="closeBurgerMenu" />
+        @toggleConnectModal="toggleWalletConnectModal" />
```

Comment:
Could you pls check if this change would break something? I have seen that the `closeBurgerMenu` event was listened at several places. How about providing both two kinds of events.
<img width="914" alt="image" src="https://user-images.githubusercontent.com/31397967/230752074-0b773ee4-4906-4450-abc8-7d526e928d76.png">

Reply:
closeBurgerMenu is also being called inside toggleWalletConnectModal, so its all working fine